packages/@okta/vuepress-site/docs/guides/protect-your-api/main/index.md

```diff
@@ -30,27 +30,39 @@ A [custom authorization server](/docs/concepts/auth-servers/#custom-authorizatio
 
 ## Set up Okta
 
-Set up your [Okta org](/docs/concepts/okta-organizations/). The CLI is the quickest way to work with your Okta org, so we recommend using it for the first few steps. If you don't want to install the CLI, you can [manually sign up for an org](https://developer.okta.com/signup/) instead. We provide non-CLI instructions along with the CLI steps below.
+Set up your developer account and [Okta org](/docs/concepts/okta-organizations/). The Okta command-line interface (CLI) is the quickest way to do this. Alternatively, you can [manually sign up for a developer account](https://developer.okta.com/signup/).
 
-1. Install the Okta command-line interface: [Okta CLI](https://cli.okta.com/).
-2. If you don't already have a free Okta developer account, create one by entering `okta register` on the command line.
-3. Make a note of the Okta Domain as you need that later.
-4. **IMPORTANT:** Set the password for your Okta developer org by opening the link that's shown after your domain is registered. Look for output similar to this:
+1. Install [Okta CLI](https://cli.okta.com/).
+1. Open your terminal.
+1. Run `okta register`, and enter your first name, last name, email address, and country.
+1. Click or tap **Activate** in the account activation email that is sent to the email address that you gave.
+
+   > **Tip**: If you don't receive the confirmation email sent as part of the creation process, check your spam filters for an email from `noreply@okta.com`
+
+1. After your domain is registered, look for output similar to this:
 
    ```txt
    Your Okta Domain: https://dev-xxxxxxx.okta.com
    To set your password open this link:
    https://dev-xxxxxxx.okta.com/welcome/xrqyNKPCZcvxL1ouKUoh
    ```
 
-   > **Note**: If you don't receive the confirmation email sent as part of the creation process, check your spam filters for an email from `noreply@okta.com`
+1. Set the password for your org by opening the link and following the instructions. Your Okta domain is returned, similar to the following:
 
-5. Connect to your Okta developer org if you didn't create one in the last step (successfully creating an Okta org also signs you in) by running the following command. You need the URL of your org &mdash; which is your [Okta domain](/docs/guides/find-your-domain/) with `https://` prepended &mdash; and an [API/access token](/docs/guides/create-an-api-token/):
-
-   ```bash
-   okta login
+   ```txt
+   New Okta Account created!
+   Your Okta Domain: https://dev-xxxxxxx.okta.com
    ```
 
+1. Make a note of your Okta domain. Use it wherever you see `${yourOktaDomain}` in the lab.
+
+> **Note**: If you're using an existing org, check API Access Management is enabled: Open your Admin Console, go to **Security** > **API**, and verify an **Authorization Servers** tab is present. If not, you can:
```

Comment:
"check API Access..." > "verify that API Access..."
"verify an **Auth..." > "verify that an **Auth.."
"If not, you can:" > "If not, follow these steps:" (avoid sentence fragment)